### PR TITLE
Remove brightcove and replace with anvato

### PR DESCRIPTION
**Pull request by critmcdonald**
This removes the brightcove player and replaces it in anvato.

- two new partials, one for full width and one for block width video
- new helper to help manage the video ad based on the site
- Readme updated about the proper frontmatter to use to use these videos
- brightcove files removed

There is a chance CMGt will come back about the ads, as they are checking if the [source that I drew them from](https://docs.google.com/document/d/1n6rwIdyyfiMpAePBtWU9PFkNUOs6wT6OZG4WvTuFIkQ/edit#heading=h.ouqmbi1ju2oc) is sufficient.

### CODE REVIEW

File: partials/video-full-anvato.hbs [hunk 1]
```diff
@@ -0,0 +1,35 @@
+<div class="photo-full">
+<div id="{{anvatoInstance}}"></div>
+
+<h3>{{ video-ad anvatoAd }}</h3>
```
Comment (danhillreports) [inline partials/video-full-anvato.hbs]: Leftover from debugging?

Comment (critmcdonald) [inline partials/video-full-anvato.hbs]: Yes ... I need to fix that.

File: helpers/video-ad.js [hunk 2]
```diff
@@ -0,0 +1,35 @@
...
+  }
+  // default to statesman
+  else {
+    var adSite = 'Austin_NP/aas_web_default'
+  }
+  // this is passed back to the template for the anvato
+  var code = 'dfp: {' +
+              'clientSide: {' +
+                  'adTagUrl:\'https://pubads.g.doubleclick.net/gampad/ads?sz=400x300&iu=/12523293/' + adSite + '&cmsid=17693&vid=ANV_ANV[[VIDEO_ID]]&ciu_szs&impl=s&gdfp_req=1&env=vp&output=vast&unviewed_position_start=1&url=[referrer_url]&description_url=[description_url]&correlator=[timestamp]\'' +
+              '}' +
```
Comment (danhillreports) [inline helpers/video-ad.js]: are those bracketed sections supposed to be populated by anything?

> vid=ANV_ANV[[VIDEO_ID]]&ciu_szs&impl=s&gdfp_req=1&env=vp&output=vast&unviewed_position_start=1&url=[referrer_url]&description_url=[description_url]&correlator=[timestamp]

Comment (critmcdonald) [inline helpers/video-ad.js]: I'll be honest ... I'm not sure. It's the same way on hookem: https://pubads.g.doubleclick.net/gampad/ads?sz=400x300&iu=/12523293/Austin_NP/aas_hookem_default/pre_roll&impl=s&gdfp_req=1&env=vp&output=vast&unviewed_position_start=1&url=[referrer_url]&description_url=[description_url]&correlator=[timestamp]

Comment (critmcdonald) [inline helpers/video-ad.js]: I've asked CMG about those values, but will continue on until I hear otherwise.

**Pull request merged**